### PR TITLE
Add multilined note with \n

### PR DESCRIPTION
**Pull request by alexandrebarbaruiva**
#73

### CODE REVIEW

File: wunderline-add.js [hunk 1]
```diff
@@ -139,6 +139,9 @@ function main() {
           });
         },
         function(task, cb) {
+          while(app.note.indexOf("\\n") !== -1) {
```
Comment (Thor77) [inline wunderline-add.js]: I don't really like this way of doing a string replacement,
please consider using a RegEx instead: `app.note.replace(/\\n/g, "\n")` [ref](https://stackoverflow.com/questions/1144783/how-to-replace-all-occurrences-of-a-string-in-javascript) should provide the same result.
(If you don't want to deal with such a minor change, I can just merge your PR and do it afterwards if you want me to)

**Review by Thor77 — CHANGES_REQUESTED** [review]

File: wunderline-add.js (hunk 1 above)
Comment (alexandrebarbaruiva) [reply wunderline-add.js]: I'll do it! I'm trying to learn as much as possible with javascript

**Comment by Thor77** [comment]
Thanks :+1:

**Comment by Thor77** [comment]
Published as v4.8.0 :tada:

**Pull request merged**